### ISSUE TITLE
_include, _revinclude seem to be ignored

### ISSUE THREAD

**Issue by mmtrun** [omoponfhir/omoponfhir-main#12]
Although the demo interface includes examples of both _include and _revinclude, neither seem to affect the results of the query. The following queries, for instance, produce exactly the same output:

```
<API>/omoponfhir4/fhir/Encounter?_id=34
<API>/omoponfhir4/fhir/Encounter?_id=34&_include=Encounter:patient
<API>/omoponfhir4/fhir/Encounter?_id=34&_revinclude=*
```

Reverse chaining filters ("_has") also appear to be ignored.

Is this expected?

**Comment by myungchoi** [comment]
Yes. Those are not yet implemented and ignored. These are pushed back for its low priority in usage.
